scripts/imgtool/image.py

```diff
@@ -40,6 +40,7 @@
 
 IMAGE_MAGIC = 0x96f3b83d
 IMAGE_HEADER_SIZE = 32
+IMAGE_HEADER_SIZE_MAX = 0x400 # limited by buffer size in loader.c:boot_copy_region
```

Comment:
It might also be helpful to put a comment in loader.c where the buffer is defined that it should match this declaration here.